### PR TITLE
Add GetSchema() API

### PR DESCRIPTION
**Pull request by Pardhakeswar**
Signed-off-by: Pardhakeswar Pacha <ppacha@nvidia.com>

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -389,6 +392,10 @@ func (c *ovndb) Close() error {
 	return nil
 }
 
+func (c *ovndb) GetSchema() (libovsdb.DatabaseSchema, error) {
+        return c.client.Schema[c.db], nil
```
Comment (girishmg) [inline client.go:396]: @Pardhakeswar  I don't think you need to return error at all.. since it is going to be `nil` all the time

Comment (Pardhakeswar) [reply client.go]: Updated it @girishmg

**Comment by girishmg** [comment]
/lgtm

@hzhou8 @noah8713 @vtolstov the background behind this is that in `ovn-k8s` we want to make sure that the OVN backend supports a table before we call APIs. Having this API gives the client a chance to make such decisions.

**Comment by noah8713** [comment]
Thanks @Pardhakeswar @girishmg  . Looks neat. /lgtm.
Seems https://github.com/eBay/go-ovn/pull/133 can re-use this as it validates current table with getschema.

Will wait a bit if @hzhou8 @vtolstov have any other suggestions. Can be merged accordingly.

**Pull request merged**